lib/groups.js

```diff
@@ -7,9 +7,9 @@
 const superagent = require('superagent');
 const logger = app.locals.logger;
 
-const uri = process.env.GAMBIT_GROUPS_API_URI;
-const apiKey = process.env.GAMBIT_GROUPS_API_KEY;
-const apiHeaderName = 'x-gambit-group-api-key';
+const uri = process.env.MESSAGING_GROUPS_API_URI;
```

Comment:
Can we namespace this env variable with `DS_MESSAGING_GROUPS` to match the other DS API env variables?  If we ever add FB Messenger as a supported client, this could get confusing.